resources/public/api/conf/1.0/application.raml

```diff
@@ -54,4 +54,14 @@ uses:
               description: 'An identifier for the income and expenditure period summary.<p>Must conform to the regular expression `^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$`'
               example: 4557ecb5-fd32-48cc-81f5-e6acd1099f3c
           get: !include foreign_property_period_summary_retrieve.raml
-          put: !include foreign_property_period_summary_amend.raml
\ No newline at end of file
+          put: !include foreign_property_period_summary_amend.raml
+      /annual:
+        (annotations.group):
+          name: Foreign Property Annual Submission
+          description: Resources relating to an individual's foreign property Annual Submission
+        /{taxYear}:
+          uriParameters:
+            taxYear:
+              description: 'The tax year to which the data applies.The start year and end year must not span two tax years. No gaps are allowed, for example, 2019-21 is not valid. The minimum tax year is 2021-22.'
```

Comment:
Minimum tax year is different on retrieve and amend endpoints - not sure which is right, need to chase that up. One of them needs changing (not sure which one though).